2.train-VAE/2.fetch_hyper_params.ipynb

```diff
@@ -67,13 +67,13 @@
      "output_type": "stream",
```

Comment:
<p>some comments here explaining what this plot is might be helpful. I struggle with explaining these plots too, so this might be something that we can both talk through.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>